src/d3a/d3a_core/simulation.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, setup_module_name: str, simulation_config: SimulationConfig =
     def _set_traversal_length(self):
         no_of_levels = self._get_setup_levels(self.area) + 1
         num_ticks_to_propagate = no_of_levels * 2
-        ConstSettings.GeneralSettings.MAX_OFFER_TRAVERSAL_LENGTH = int(num_ticks_to_propagate)
+        ConstSettings.GeneralSettings.MAX_OFFER_TRAVERSAL_LENGTH = 2
```

Comment:
If you use the default value, you can omit this line.

Reply:
Or omit the setting in const.py.

Reply:
Yes but the purpose of this method is to calculate MAX_OFFER_TRAVERSAL_LENGTH which is not used any where

Reply:
Yes, and to inform the user how long the bids and offers need to propagate. If this is not needed any more, feel free to remove this function.